operations/mimir/config.libsonnet

```diff
@@ -551,6 +551,8 @@
     gossip_member_label: 'gossip_ring_member',
     // Labels that service selectors should not use
     service_ignored_labels:: [self.gossip_member_label],
+
+    create_service_monitor: false,
```

Comment:
[nit] Can we rename it to `service_monitor_enabled` to keep some consistency with existing bool config options, please?